pkg/ansible/runner/runner_test.go

```diff
@@ -63,111 +51,88 @@ func checkCmdFunc(t *testing.T, cmdFunc cmdFuncType, role, playbook string, watc
 }
 
 func TestNew(t *testing.T) {
+	cwd, err := os.Getwd()
+	if err != nil {
+		t.Fatalf("Unable to get working director: %v", err)
+	}
+	validPlaybook := filepath.Join(cwd, "testdata", "playbook.yml")
+	validRole := filepath.Join(cwd, "testdata", "roles", "role")
 	testCases := []struct {
-		name  string
-		watch watches.Watch
+		name      string
+		gvk       schema.GroupVersionKind
+		playbook  string
+		role      string
+		finalizer *watches.Finalizer
```

Comment:
Why is required change the watches and add the finalizer to add support to users defined the verbose level of the ansible container?

Reply:
Since I am creating the `watch` below, I need all of the things that could be meaningfully define on a watch to be a part of each test case.